rest_live/signals.py

```diff
@@ -29,15 +27,14 @@ async def send_model_update(
         channel_layer = get_channel_layer()
         group_name = get_group_name(model_label, group_key, group_key_prop)
 
-        @sync_to_async
-        def get_serializer_data():
-            return camelize(serializer.data)
+        content = {
+            "model": model_label,
+            "instance": camelize(serializer.data),
+            "action": action,
+            "group_key_value": group_key,
+        }
 
-        serializer_data = await get_serializer_data()
-
-        content = {"model": model_label, "instance": serializer_data, "action": action, "group_key_value": group_key}
-
-        await channel_layer.group_send(
+        async_to_sync(channel_layer.group_send)(
```

Comment:
The ORM isn't async yet, so this might throw errors I think. At least that's why the `async_to_sync` decorator is recommended. But we can see if it works in production on OHQ and just run with it. Just something to keep in mind before django 3.2 comes out with a fully async path for the ORM.

Reply:
With these changes, the entirety of send_model_update is synchronous, so from my understanding, it does all the db calls synchronously then uses `async_to_sync` to send the updates through channels.

I'm pretty confident that the issue we were having earlier is [related to this](https://channels.readthedocs.io/en/latest/topics/databases.html#database-sync-to-async) but I think that this way of changing things will work better than using the `db_to_sync` method on lines 54 and 58